src/Robo/Tasks/RoboFile.php

```diff
@@ -22,6 +22,45 @@ class RoboFile extends Tasks {
   public function __construct() {
   }
 
+  /**
+   * Adjust the composer.json and info.yml files back to dev after a release.
+   *
+   * @param string $old_semver
+   *   Recently released semver tag.
+   * @param string $directory
+   *   Directory where the module is to be updated.
+   *
+   * @command back-to-dev
+   * @usage vendor/bin/sws-caravan back-to-dev 8.1.0 /var/www/stanford_module
+   * @usage vendor/bin/sws-caravan back-to-dev ${CIRCLE_TAG} ${CIRCLE_WORKING_DIRECTORY}
+   */
+  public function backToDev($old_semver, $directory) {
+    list($major, $minor, $point) = explode('.', $old_semver);
+    $info_yamls = $this->rglob("$directory/*.info.yml");
```

Comment:
This is probably safe. I cannot think of anything else that uses this convention.